src/backend/devices/formula.py

```diff
@@ -8,6 +8,10 @@ def voltage(value):
     @staticmethod
     def adc_to_volts(adc):
         return (adc / 4095.000) * 5.0
+    
+    @staticmethod
+    def adc16_to_volts(adc):
+        return (adc / 51)
```

Comment:
I think is interesting to know where 51 comes from, like the formula above, maybe is not the best performance to have more operations but anyway we will not feel anything even in a raspberry.

Reply:
This is the way I've always handled 5V ADC, it converts 0-255 straight to a 5V range. I can see where the number could get ugly on a lesser processor though

Reply:
ok, then there's still something confusing here.
0-255 means 8 bit then the formula should be named `adc8_to_volts`.
Also that means that 51 comes from 255/5 which could be translated into `return (adc / 255) * 5` wich I think that gives more hints about what we are doing (rule of three).

Reply:
Yes sorry, that's correct. Brain fart. I think I initially called it adc8 and then changed it at some point. Either way, the way you spelled it out makes more sense to someone who might read through the code later.

Reply:
Changed adc16 to adc8 and updated formula.